modules/data_policy_export/data_policy_export.install

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the data_policy_export module.
+ */
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Implements hook_install().
+ */
+function data_policy_export_install() {
+  // Set our adjusted views data as the new config.
+    $config_file = drupal_get_path('module', 'data_policy_export') . '/config/update/data_policy_export_view.yml';
```

Comment:
We need to change the approach as this may totally override the changes by others. So, we need to fetch the current configurations, add our field in the array and save it back to DB. 

While uninstalling, we need to remove our changes and save the rest back to DB.

Reply:
This is fixed now; on install we import our config and on uninstall, we remove it again.